gitpod-io/gitpod
# [dashboard] Always start with options

## Description
Introduces a new CreateWorkspacePage on which the options are shown and can be changed.

The page is only activated for users who are in team-only mode already (change your email to `toa@gitpod.io` in user settings and reload the dashboard to activate it). There is also a feature flag that can enable it for other users (currently false everywhere). 

This also moves /new to /projects/new so /new can be used for creating new workspaces.

**Without the feature flag there will be no difference in starting workspaces, with one exception: `/new` works for everybody**

## Related Issue(s)

Fixes #16362

## How to test


1) Try starting workspaces in different ways. Should all work as usual.
2) Change the user email to `toa@gitpod.io` and reload the browser to active org only attribution.
3) All workspace creations now go to a new create workspace page. 

## Release Notes

```release-note
New workspace creation page
```

## Documentation


## Build Options:

      Experimental feature to run the build with GitHub Actions (and not in Werft).
      leeway-target=components:all
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

</details>

<details>
<summary>Installer Options</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
